pkg/redshift/driver/rows_test.go

```diff
@@ -383,6 +383,19 @@ func Test_convertRow(t *testing.T) {
 			expectedValue: `2021-07-15 14:00:00 +0000 UTC`,
 			Err:           require.NoError,
 		},
+		{
+			name: "null",
+			metadata: &redshiftdataapiservice.ColumnMetadata{
+				Name:     aws.String("id"),
+				TypeName: aws.String(REDSHIFT_INT8),
```

Comment:
This `TypeName` field can be removed in this test case.
(It might not be a realistic representation of that data, but that documents that this field is irrelevant, i.e. the code doesn't even use it.)